### PR TITLE
Elasticsearch backup and generic backup page

### PR DESCRIPTION
**Pull request by jansaidl**
#### What Type of Change is this?

- [x] New Page
- [ ] Minor Fix
- [ ] Minor Improvement
- [ ] Major Improvement

#### Description (required)

* add ElasticSearch backup section
* add generic backup page

#### Related issues & labels (optional)

- /closes #

### CODE REVIEW

File: apps/docs/sidebars.js [hunk 1]
```diff
@@ -97,6 +97,15 @@ module.exports = {
       },
       className: 'homepage-sidebar-item',
     },
+    {
+      type: 'doc',
+      id: 'features/backup',
+      label: 'Backup',
+      customProps: {
+        sidebar_icon: 'currency-dollar',
```
Comment (vankovap) [inline apps/docs/sidebars.js]: nejakou peknejsi ikonku prosim, treba `'archive-box'`

File: apps/docs/content/features/backup.mdx [hunk 2]
```diff
@@ -0,0 +1,50 @@
+---
+title: Backup data
+description: Learn how you can work with backups in Zerops works.
+---
+
+Zerops provides data backup for certain services. The data is stored in S3 storage.
```
Comment (vankovap) [inline apps/docs/content/features/backup.mdx]: mozna bych trochu vic specifikovala, ktery services to aktualne jsou

**Review by vankovap — CHANGES_REQUESTED** [review]
muzes prosim projit ten task na backup a pridat/upravit podle toho? dost toho tu chybi

File: apps/docs/content/features/backup.mdx (hunk 2 above)
Comment (jansaidl) [reply apps/docs/content/features/backup.mdx]: Doplnil jsem vetu,z e to je v detailu service.

**Comment by jansaidl** [comment]
> muzes prosim projit ten task na backup a pridat/upravit podle toho? dost toho tu chybi

Neco sem doplnil, neco vynechal s tim, ze to nemame.

**Review by vankovap — APPROVED** [review]

**Pull request merged**